<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepartmentEditDialog</class>
 <widget class="QDialog" name="DepartmentEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DepartmentEditDialog</string>
  </property>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDeptId">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Dept id</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDeptName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Dept name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDeptId">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDeptName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
